tests/conftest.py

```diff
@@ -56,3 +66,73 @@ def minimal_record(minimal_input_record):
     record["publication_date"] = date.today().isoformat()
     record["_publication_date_search"] = date.today().isoformat()
     return record
+
+
+@pytest.fixture()
+def users(app, db):
+    """Create users."""
+    def dump_user(user):
+        """User object to dict."""
+        return {
+            'email': user.email,
+            'id': user.id,
+            'password': password
+        }
+    password = '123456'
+    with db.session.begin_nested():
+        datastore = app.extensions['security'].datastore
+        # create users
+        hashed_password = hash_password(password)
```

Comment:
I used that just because the `invenio-accounts.rest.auth_login` tries to match the password with the hashed value i.e it forces to create a user with a hashed password 😅  But, I think as we write more tests, the need of some helper functions, as the one you suggested, will arise...